test/cmdlineTests/optimizer_inliner_dynamic_reference/output

```diff
@@ -194,7 +194,7 @@ sub_0: assembly {
         /* "#utility.yul":406:415   */
       add
       swap1
-        /* "#utility.yul":244:421   */
+        /* "#utility.yul":196:421   */
```

Comment:
This now references the full function instead of the last block (or even statement?)